scripts/create-admin-user.sh

```diff
@@ -27,7 +27,7 @@ fi
 while [ "0" -ne "$(kubectl exec deployment/reana-db -- pg_isready -U reana -h 127.0.0.1 -p 5432 &> /dev/null && echo $? || echo 1)" ]
 do
     echo "Waiting for deployment/reana-db to be ready..."
-    sleep 5
+    sleep 10
```

Comment:
BTW sleep 10 should not be helping much here, since this check is done within a big loop waiting for pod to occur, we can just as well be checking every 5 seconds...

Reply:
It is not because of this fix, it is because during the many times I've run the command, I got many connection errors because of Posgres not being ready, so 5 seconds for my setup/laptop is not enough, I will create a separate commit.

Reply:
Hmm, for me 5 secs seems fine, but we may indeed increase a bit if necessary for some hardware.